src/Sylius/Bundle/ApiBundle/Resources/config/serialization/OrderItem.xml

```diff
@@ -52,6 +52,12 @@
             <group>shop:order_item:read</group>
             <group>shop:cart:read</group>
         </attribute>
+        <attribute name="originalUnitPrice" serialized-name="originalPrice">
```

Comment:
I know that `originalPrice` was already used, but I would be for changing it to `originalUnitPrice` and add a proper note to UPGRADE-API-1.11 file (or not, if it was added in 1.11)
```suggestion
        <attribute name="originalUnitPrice">
```